media/src/main/java/edu/artic/media/audio/AudioPlayerService.kt

```diff
@@ -423,7 +423,7 @@ class AudioPlayerService : DaggerService(), PlayerService {
     }
 
     override fun stopPlayer() {
-        analyticsTracker.reportEvent(EventCategoryName.PlayBack, AnalyticsAction.playbackInterrupted, (currentTrack as BehaviorSubject).value?.title.orEmpty())
+        analyticsTracker.reportEvent(EventCategoryName.PlayBack, AnalyticsAction.playAudioTourStop, (currentTrack as BehaviorSubject).value?.title.orEmpty())
```

Comment:
Hold on, isn't that the event for when you play the audio at a given tour stop? I don't recall seeing anything in the ticket about removing `playbackInterrupted`.

Reply:
Check above in this class `playbackInterrupted` is still called when the user switches tracks. Player stopped is indicating the user stop the playback all together

Reply:
Ah, I missed that.

It _is_ weird that the new event here has the word `tour` in it....you're sure that's the right constant?

Reply:
hmm you have a point there. is there a way to know if we are in a tour currently?

Reply:
That's tracked in the `TourProgressManager` singleton. It doesn't help that we have both two distinct concepts here:

1. 'stopping a tour'
2. 'reaching a stop on the tour'

Reply:
Fwiw the requirements say

    Triggered when the user plays an audio track by pressing play on a tour stop.

Reply:
sigh thanks